benchexec/tools/muval.py

```diff
@@ -0,0 +1,89 @@
+"""
```

Comment:
Please add a license and copyright header using the [REUSE format](https://reuse.software/), like we have in all files in the repository.